tool/tsh/tsh.go

```diff
@@ -736,6 +763,22 @@ func Run(args []string, opts ...cliOption) error {
 	return trace.Wrap(err)
 }
 
+func loadConfig(homePath string) (*TshConfig, error) {
+	confPath := filepath.Join(profile.FullProfilePath(homePath), tshConfigPath)
+	configFile, err := os.Open(confPath)
+	if err != nil {
+		if trace.IsNotFound(err) {
+			// if there is no config file, continue without logging.
+			return nil, nil
```

Comment:
Let's avoid returning `nil, nil`. Instead, just return the error as-is and check for trace.IsNotFound on the caller side.